WordPress/src/main/res/layout/post_list_item_compact.xml

```diff
@@ -0,0 +1,115 @@
+            android:layout_marginEnd="@dimen/margin_extra_large"
+            android:layout_marginStart="@dimen/margin_extra_large"
+            android:layout_marginTop="@dimen/margin_large"
+            android:layout_toStartOf="@+id/image_featured"
+            android:ellipsize="end"
+            android:fontFamily="serif"
+            android:maxLines="1"
+            android:textAlignment="textStart"
+            android:textColor="@color/neutral_800"
+            android:textSize="@dimen/text_sz_large"
+            android:textStyle="bold"
+            app:layout_goneMarginTop="@dimen/margin_extra_large"
+            tools:text="Testing title Testing title Testing title Testing title Testing title Testing title Testing title Testing title"/>
+
+        <org.wordpress.android.widgets.WPTextView
+            android:id="@+id/date"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_alignParentStart="true"
+            android:layout_below="@+id/title"
+            android:layout_marginBottom="6dp"
+            android:layout_marginStart="@dimen/margin_extra_large"
+            android:layout_marginTop="@dimen/margin_medium"
+            android:ellipsize="end"
+            android:fontFamily="serif"
+            android:maxLines="1"
+            android:textColor="@color/neutral_500"
+            android:textSize="@dimen/text_sz_medium"
+            tools:text="99 days ago"/>
+
+        <org.wordpress.android.widgets.WPTextView
+            android:id="@+id/statuses_label"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/title"
+            android:layout_marginEnd="@dimen/margin_medium"
+            android:layout_marginStart="@dimen/margin_medium"
+            android:layout_marginTop="@dimen/margin_medium"
+            android:layout_toEndOf="@+id/date"
+            android:layout_toStartOf="@+id/image_featured"
+            android:ellipsize="end"
+            android:fontFamily="serif"
+            android:maxLines="1"
+            android:textAlignment="textStart"
+            android:textColor="@color/warning_500"
+            app:layout_goneMarginBottom="@dimen/margin_extra_large"
+            tools:text="Private · Local Changes"/>
+
+
+        <ImageView
+            android:id="@+id/image_featured"
+            android:layout_width="@dimen/posts_list_compact_image_size"
+            android:layout_height="@dimen/posts_list_compact_image_size"
+            android:layout_centerVertical="true"
+            android:layout_marginBottom="@dimen/margin_large"
+            android:layout_marginEnd="@dimen/margin_small"
+            android:layout_marginStart="@dimen/margin_small"
+            android:layout_marginTop="@dimen/margin_large"
+            android:layout_toStartOf="@+id/more_button"
+            android:background="@color/red_0"
```

Comment:
Do we intentionally set the background to red color?

Reply:
Ah - I did for reasons of laying the view out and seeing the preview (since I couldn't see white-on-white) but I'll change it to transparent now I'm done

Reply:
I think we could just remove the background attribute, but it doesn't really matter. 👍